views/lexicon/index.html.twig

```diff
@@ -37,6 +37,7 @@
 {% set tab_loading %}{% include   'lexicon/tabs/loading.html.twig' %}{% endset %}
 {% set tab_message %}{% include   'lexicon/tabs/message.html.twig' %}{% endset %}
 {% set tab_modals %}{% include    'lexicon/tabs/modals.html.twig' %}{% endset %}
+{% set tab_pagination %}{% include 'lexicon/tabs/pagination.html.twig' %}{% endset %}
```

Comment:
Spacing looks a bit hotch potch, could this be spaced either with 1 space, or tabbed nicely inline